### DISCUSSION TITLE
Waybar time/date

### DISCUSSION THREAD

**Discussion by Rukius1** [prasanthrangan/hyprdots#1782] [Q&A] [answered]
Is there a way to always have the time/date version as the default option on the waybar?

**Comment by kRHYME7** [comment, marked as answer]
First duplicate this file 
` ~/.config/waybar/modules/clock.jsonc ` and rename it to something ` ~/.config/waybar/modules/clock#myclock.jsonc ` ( I named mine clock#user.jsonc) note that you should use `#` then the suffix

` cp ~/.config/waybar/modules/clock.jsonc ~/.config/waybar/modules/clock#user.jsonc ` 

Now edit this duplicated module  ( in your case you just want to swap the format and format-alt )

![image](https://github.com/user-attachments/assets/dead1164-a05f-40af-aeed-7aecba64cccb)


Now to make this layout take effect

edit your ` ~/.config/waybar/config.ctl ` then change the ` clock ` into ` clock#user `  then toggle your waybar layout using a keybind. It should change the time. 

  
Note you can directly change the ` ~/.config/waybar/modules/clock.jsonc ` but if you want to fetch for updates in from repo there is a chance to overwrite your changes. by tagging the modules with #tag it acts a separate file and the script will ignore it. 


before: 
![image](https://github.com/user-attachments/assets/7520d892-ac02-48d5-8da9-0073a71700e9)

after 

![image](https://github.com/user-attachments/assets/ee07bbab-500c-42cb-9735-a8d89171101c)
